### PR TITLE
Allow for users not logged in to view business pages

### PR DESCRIPTION
**Pull request by eash130**
I accidentally pushed this change with the typo CL, it's an extra step so that if a user that isn't logged into the website tries to navigate to a business page, they won't be encountered with a null pointer exception.

If it looks good, then I'll just close the PR.

### CODE REVIEW

File: capstone/src/main/java/com/google/sps/servlets/business/BusinessServlet.java [hunk 1]
```diff
@@ -75,10 +75,12 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
           "The business you were looking was not found in our records!");
       return;
     }
+    // SEE BELOW
     String id = "";
     if (userService.getCurrentUser() != null) {
       id = userService.getCurrentUser().getUserId();
     }
+    // SEE ABOVE
```
Comment (lukasberglund) [inline capstone/src/main/java/com/google/sps/servlets/business/BusinessServlet.java:83]: These comments confuse me.

Comment (eash130) [reply capstone/src/main/java/com/google/sps/servlets/business/BusinessServlet.java:83]: the edit that i made before was on lines between the comments. I accidentally pushed this with another PR.

Comment (eash130) [reply capstone/src/main/java/com/google/sps/servlets/business/BusinessServlet.java:83]: If the code in between looks fine, then I'll just close this PR

Comment (eash130) [reply capstone/src/main/java/com/google/sps/servlets/business/BusinessServlet.java:83]: Looking at it now, I'm not sure why. I only added the conditional.

**Pull request merged**